app/views/users/_user.html.erb

```diff
@@ -1,7 +1,7 @@
 
 <div class="d-flex gap-3 pb-4" >    
-    <div class="flex-grow-1  bg-success image-container" >
-       <img src="<%=@user.photo%>" alt="" class="image">
+    <div class="flex-grow-1  bg-success" >
+       <img src="#" alt=""> 
     </div>
     <div class="flex-grow-1 w-50 bg-primary py-4 px-4 d-flex flex-column gap-3" >      
       
```

Comment:
- [ ] Line 9 of this file has the same error as above. Should be `post_counter`.

  ![image](https://github.com/Marcraphael12/rails-blog-app/assets/36542986/8a9db935-6e2a-44ef-abf1-27a11a631cbe)